omok/main.py

```diff
@@ -0,0 +1,18 @@
+import configparser
```

Comment:
maybe this should be outside the pkg `omok`

Reply:
Maybe not - https://stackoverflow.com/questions/193161/what-is-the-best-project-structure-for-a-python-application.